helmfile/overrides/notify/celery.yaml.gotmpl

```diff
@@ -43,7 +43,7 @@ nodes:
     deployment:
       deploymentBeatEnabled: {{ if eq .Environment.Name "production" }} true {{ else if eq .Environment.Name "staging" }} true {{ else }} true {{ end }}
```

Comment:
Could we put the value to false here? different way and same result?

Reply:
Yes, with this way the deployment won't deploy... which would not validate the code... I'd rather deploy with 0 to validate that the code at least works.